switchlink/switchlink_neigh.c

```diff
@@ -105,7 +108,6 @@ void switchlink_process_neigh_msg(const struct nlmsghdr *nlmsg, int msgtype) {
         break;
       }
       default:
-        krnlmon_log_debug("neigh: skipping attr %d\n", attr_type);
```

Comment:
Am I correct in assuming that you're removing the krnlmon_log_debug() call to cut down on logger noise?

There's certainly nothing remarkable about a message that contains attributes we don't care about.

Reply:
Yes.